ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRaoTool.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.search_tree_rao;
+
+import com.farao_community.farao.data.crac_api.Crac;
+
+import com.farao_community.farao.data.crac_io_api.CracImporters;
+import com.farao_community.farao.ra_optimisation.RaoComputationResult;
+import com.farao_community.farao.rao_api.RaoParameters;
+import com.google.auto.service.AutoService;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.import_.Importers;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.tools.Command;
+import com.powsybl.tools.Tool;
+import com.powsybl.tools.ToolRunningContext;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ */
+@AutoService(Tool.class)
+public class SearchTreeRaoTool implements Tool {
+
+    private static final String CASE_FILE_OPTION = "case-file";
+    private static final String CRAC_FILE_OPTION = "crac-file";
+    private static final String OUTPUT_FILE_OPTION = "output-file";
+    private static final String OUTPUT_FORMAT_OPTION = "output-format";
+
+    @Override
+    public Command getCommand() {
+        return new Command() {
+
+            @Override
+            public String getTheme() {
+                return "Computation";
+            }
+
+            @Override
+            public String getDescription() {
+                String description = "Run SearchTreeRao Computation";
+                return description;
```

Comment:
You can directly return  "Run SearchTreeRao Computation" without assigning it to a variable.

Reply:
ok